allenact/algorithms/onpolicy_sync/losses/ppo.py

```diff
@@ -97,11 +101,24 @@ def add_trailing_dims(t: torch.Tensor):
             )
 
         # noinspection PyUnresolvedReferences
-        return {
-            "value": (value_loss, self.value_loss_coef),
-            "action": (action_loss, None),
-            "entropy": (dist_entropy.mul_(-1.0), self.entropy_coef),  # type: ignore
-        }
+        return (
+            {
+                "value": (value_loss, self.value_loss_coef),
+                "action": (action_loss, None),
+                "entropy": (dist_entropy.mul_(-1.0), self.entropy_coef),  # type: ignore
+            },
+            {
+                "ratio": float(ratio.mean().item()),
+                "ratio_clamped": float(clamped_ratio.mean().item()),
+                "ratio_used": float(
+                    torch.where(cast(torch.Tensor, use_clamped), clamped_ratio, ratio)
+                    .mean()
+                    .item()
+                ),
```

Comment:
Can we return these without the `.mean().item()` here and then do the `.mean().item()` in the `loss` function? This would better mirror how the values in the other dictionary are being handled.

Reply:
👍